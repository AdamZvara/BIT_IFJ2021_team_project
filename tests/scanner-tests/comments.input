--single line comment      
while
-- 123489 do while end 

--[ still single ]line comment
do
--[ single line comment [[ 
else

--[[ multi 
    line
    comment ]] local
--[[ multi
    line][] not ending
]] number

--[[ multi 
	line -- single line --]]
