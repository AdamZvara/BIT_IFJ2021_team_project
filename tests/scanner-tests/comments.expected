TOK_KEYWORD : while
TOK_KEYWORD : do
TOK_KEYWORD : else
TOK_KEYWORD : local
TOK_KEYWORD : number
